### ISSUE TITLE
Finger mass

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10842]
**Preferred term label:**
Finger mass

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lump on the finger; bump on the finger; finger growth; nodule on finger; finger tumor; finger cyst

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0001167

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Digital mucous cyst
Digital artery aneurysm
Giant cell tumor of the tendon sheath (GCTTS)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001163
- label: Finger mass
- parents: [Abnormal finger morphology (HP:0001167)](https://hpo.jax.org/app/browse/term/HP:0001167)
- pmid: 
- def: An abnormal enlargement or swelling localized to a finger.
- comment: The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10842
- synonyms: Finger growth(NONE)|Lump on finger(NONE)|Bump on finger(NONE)
